CollAction/Views/Projects/ThankYouCommit.cshtml

```diff
@@ -13,10 +13,9 @@
     <div class="row">
         <div id="thank-you-commit-box" class="col-xs-12 col-md-6 col-md-offset-3">
             <div id="close-button-container">
-                <a id="close-button" asp-action="Details" asp-route-id="@Model.ProjectId">
-                    <object id="close-button-img" type="image/svg+xml" data="/images/close-button-friesland-blue.svg">
-                        @*Close*@
-                        Sluiten
+                <a id="close-button" asp-action="Details" asp-route-id="@Model.ProjectName">
+                    <object id="close-button-img" type="image/svg+xml" data="/images/close-button.svg">
+                        Close
```

Comment:
You should keep "Sluiten" if it's going to be merged to Friesland. And keep close-button-friesland-blue.svg.

Reply:
Fixed in 0544b69